sw/device/tests/sim_dv/all_escalation_resets_test.c

```diff
@@ -812,9 +815,12 @@ static void execute_test(const dif_aon_timer_t *aon_timer) {
       fault_checker = fc;
     } break;
     // TODO add mechanism to inject kTopEarlgreyAlertIdLcCtrlFatalProgError by
-    // forcing otp_prog_err_o from lc_ctrl_fsm and
-    // kTopEarlgreyAlertIdLcCtrlFatalStateError using sparse fsm.
-    // alerts, and corresponding CSR bit to check.
+    // forcing otp_prog_err_o from lc_ctrl_fsm
+    case kTopEarlgreyAlertIdLcCtrlFatalStateError: {
```

Comment:
I am not sure this can even be checked in SW, since forcing the LC_CTRL FSM into a fatal error state will disable the CPU immediately (it behaves similarly to escalation or SCRAP).